packages/airnode-deployer/src/infrastructure/aws.ts

```diff
@@ -35,8 +34,8 @@ export const getAirnodeBucket = async (cloudProvider: AwsCloudProvider) => {
   return airnodeBuckets?.[0]?.Name ?? null;
 };
 
-export const createAirnodeBucket = async (cloudProvider: AwsCloudProvider) => {
-  const s3 = initializeS3Service(cloudProvider);
+export const createAirnodeBucket = async (_cloudProvider: AwsCloudProvider) => {
```

Comment:
Why not remove the argument?

Reply:
Because the argument is needed for the GCP counterpart of this function. It needs to have the same interface.